source/ngf-vk/impl.c

```diff
@@ -249,6 +251,16 @@ typedef struct ngf_image_t {
   uint32_t       nlayers;
 } ngf_image_t;
 
+// Macros for obtaining proper RenderDoc Version
+#define NGF_RENDERDOC_API_VERSION 1_6_0
+#define NGF_RENDERDOC_VERSION NGFI_EVAL_AND_PASTE(eRENDERDOC_API_Version_, NGF_RENDERDOC_API_VERSION)
+#define NGF_RENDERDOC_API NGFI_EVAL_AND_PASTE(RENDERDOC_API_, NGF_RENDERDOC_API_VERSION)
+
+typedef struct ngf_rdoc_api_t {
+    NGF_RENDERDOC_API* api;
```

Comment:
let's use the renderdoc API struct directly here, no need for the macro magic. remove `NGFI_PASTE` etc from macros.h as well.